### DISCUSSION TITLE
auth.login and auth.register routes not working

### DISCUSSION THREAD

**Discussion by pekka** [Jacobtims/filament-breezy#173] [Q&A] [answered]
Still very new to Laravel and Filament. Very grateful for a fantastic community and excellent products! 

I am building an app that has a small frontend portion (for search etc.) and a big Filament backend. Initially I wanted to use Jetstream for handling regular users, and reserve Filament only for genuine site admins. However Filament is looking so robust that this split doesn't seem to make sense.

So now I removed Jetstream and Fortify and installed Breezy. I removed Jetstream's middleware routing from `routes/web.php`. I can log in to Filament, however when I try to register a new user, I am taken to `/register` and get 

> View [auth.register] not found.

When I try `/admin/register`, I get the correct Breezy page.

I am unsure how to fix this - the only explanation I have this is a remainder from my installing and uninstalling Jetstream (which is not recommended, I know - but the project is already so big, it'd cost me a long while to migrate to a clean install...) I read something about Laravel UI but this seems to be deprecated and shouldn't be necessary for Breezy in the first place.

What would be the right way to go to fix this?

**Comment by jeffgreco13** [comment]
Did you try `php artisan route:clear`

**Reply by pekka** [reply]
I did, that did actually change things in that I started seeing Jetstream related errors. I started removing all references to Jetstream and Fortify from the app and seem to have succeeded, but now get a 404 on `/login` and `/register`, while when I try to log in to FIlament, it takes me to the correct pages that live under `/admin/login` and `/admin/register`.

I definitely have the correct thing in `config/filament.php`: 

```
 'auth' => [
        'guard' => env('FILAMENT_AUTH_GUARD', 'web'),
        'pages' => [
            'login' =>  \JeffGreco13\FilamentBreezy\Http\Livewire\Auth\Login::class,
        ],
    ],

```

and done a `php artisan config:clear` to clear up any hiccups.

Is this expected behaviour because Breezy is _supposed_ to work only in a Filament context? 

Is there a straightforward way to make `/login` and `/register` work? 

Is it as easy as adding the correct routes to `web.php`?

Appreciate the time - I realize this is a very much a corner I've painted myself into!

**Reply by jeffgreco13** [reply, marked as answer]
You are correct that Breezy ONLY works in Filament context.
Depending on your project, you can set the path to the root directory using env: `FILAMENT_PATH=/`.
But if you do need to keep `/admin` in the path as to not interfere with other parts of your app, simple create a couple redirect routes:

```php
Route::get('/login', function(){
    return redirect()->route('filament.auth.login');
});

Route::get('/register', function(){
    return redirect()->route('filament.auth.register');
});

```

**Reply by pekka** [reply]
I see I see, think I got it now, thank you!
